server/config/config.go

```diff
@@ -24,12 +24,19 @@ type DBConfig struct {
 	MigrationsPath   string
 }
 
+//GRPCConfig grpc options
+type GRPCConfig struct {
+	MaxRecvMsgSizeInMB int `default:"10"`
+	MaxSendMsgSizeInMB int `default:"10"`
```

Comment:
I was wondering what if we just set it really big value and forget about it? I see very little value in making this configurable, to be honest?